helm/cluster-cloud-director/values.schema.json

```diff
@@ -87,39 +87,56 @@
         "apiServer": {
             "properties": {
                 "certSANs": {
+                    "description": "Alternative names to encode in the API server's certificate.",
                     "items": {
+                        "title": "SAN",
                         "type": "string"
                     },
+                    "title": "Subject alternative names (SAN)",
                     "type": "array"
                 },
                 "enableAdmissionPlugins": {
+                    "description": "Comma-separated list of admission plugins to enable.",
+                    "title": "Admission plugins",
                     "type": "string"
                 },
                 "featureGates": {
+                    "description": "Enabled feature gates, as a comma-separated list.",
+                    "title": "Feature gates",
                     "type": "string"
                 }
             },
             "required": [
                 "enableAdmissionPlugins",
                 "featureGates"
             ],
+            "title": "Kubernetes API server",
             "type": "object"
         },
         "baseDomain": {
+            "title": "Base DNS domain",
             "type": "string"
         },
         "cloudDirector": {
             "properties": {
                 "org": {
+                    "description": "VCD organization name.",
+                    "title": "Organization",
                     "type": "string"
                 },
                 "ovdc": {
+                    "description": "TODO.",
```

Comment:
The description should clarify what is expected here when filling in the value. Should this be the name or ID of an OvDC the cluster gets created in?

Reply:
This is the name of the ovdc.